streamingccl: TestTenantStreamingFailback flaked on release-23.2

This flaked on an unrelated PR: https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_BazelEssentialCi/14585251?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&expandBuildChangesSection=true&expandBuildTestsSection=true
```
Failed
=== RUN   TestTenantStreamingFailback
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingFailback3608586212
    test_log_scope.go:81: use -show-logs to present logs inline
    stream_ingestion_job_test.go:205: creating tenant f
    stream_ingestion_job_test.go:216: starting replication f->g
    stream_ingestion_job_test.go:223: waiting for g@1711552198088635551.0000000000
    stream_ingestion_job_test.go:229: completing replication on g@1711552198088635551.0000000000
    stream_ingestion_job_test.go:751: tenant "f" is not accepting connections
    stream_ingestion_job_test.go:255: starting replication g->f
    stream_ingestion_job_test.go:259: waiting for f@1711552218994527871.0000000000
    stream_ingestion_job_test.go:278: completing replication on f@1711552229914718032.0000000000
    stream_ingestion_job_test.go:751: tenant "g" is not accepting connections
    stream_ingestion_job_test.go:285: starting replication f->g
    stream_ingestion_job_test.go:289: waiting for g@1711552229914718032.0000000000
    stream_ingestion_job_test.go:168:
          Error Trace:  github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamingest/stream_ingestion_job_test.go:168
                              github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamingest/stream_ingestion_job_test.go:299
          Error:        Not equal:
                        expected: 6412387353320329170
                        actual  : 4077712811438800083
          Test:         TestTenantStreamingFailback
          Messages:     fingerprint mismatch at 1711552229914718032.0000000000
    panic.go:541: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingFailback3608586212
--- FAIL: TestTenantStreamingFailback (63.76s)
```

Jira issue: CRDB-37143

--- a/pkg/ccl/streamingccl/streamingest/stream_ingestion_job_test.go
+++ b/pkg/ccl/streamingccl/streamingest/stream_ingestion_job_test.go
@@ -102,7 +102,7 @@ func TestTenantStreamingFailback(t *testing.T) {
 	defer log.Scope(t).Close(t)
 	ctx := context.Background()
 
-	skip.UnderStressRace(t, "test takes ~5 minutes under stressrace")
+	skip.WithIssue(t, 121221, "failback unsuppported on 23.2")
 
 	serverA, aDB, _ := serverutils.StartServer(t, base.TestServerArgs{
 		DefaultTestTenant: base.TestControlsTenantsExplicitly,

